src/Tribe/Integrations/WPML/Meta.php

```diff
@@ -28,6 +28,8 @@ public function hook() {
 	 * @return mixed The translated id for _EventOrganizerID & _EventVenueID or false.
 	 */
 	public function translate_post_id( $value, $object_id, $meta_key ) {
+		static $cached_values = [];
```

Comment:
Resetting a cached value set using the `static` is almost impossible. I would use `tribe_cache()` here to get hold of the singleton instance of the Cache implementation, then use its `ArrayAccess` API to store values on a per-request basis.
```php
$cache = tribe_cache();

$cache[$cache_key] = $value; // Set value in the cache.

$cached_value = $cache[$cache_key]; // Get from the cache
```

This will allow the cache to be flushed along with the rest of the WordPress cache in the context of a request, or tests.